src/tests/ftest/scripts/main.sh

```diff
@@ -38,6 +38,14 @@
 
 set -eux
 
+# check that vm.max_map_count has been configured/bumped
+if [ $(sudo sysctl -n vm.max_map_count) != 1000000 ] ; then
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]

Reply:
Ideally, we should resolve these lint issues